tests/FakeItEasy.Analyzer.CSharp.Tests/RepeatedAssertionAnalyzerTests.cs

```diff
@@ -11,6 +11,7 @@ namespace FakeItEasy.Analyzer.CSharp.Tests
     public class RepeatedAssertionAnalyzerTests : CodeFixVerifier
     {
         private const string CodeTemplate = @"using FakeItEasy;
+#pragma warning disable CS0618

```

Comment:
HA! I thought you added  pragma to the code, not a pragma to the test data. How embarrassing. Carry on!